AirCasting/Settings/SettingsView.swift

```diff
@@ -294,6 +302,39 @@ struct SettingsView: View {
              }
          }
      }
+    
+    private var calibrateMicrophone: some View {
+        Button(action: {
+            viewModel.manualMicrophoneCalibrationTapped()
+        }) {
+            VStack {
+                buttonCell(title: Strings.MicrophoneCalibration.settingsItemTitle)
+                Spacer()
+                Text(Strings.MicrophoneCalibration.settingsItemDescription)
+                    .font(Fonts.muliRegularHeading3)
+                    .foregroundColor(.aircastingGray)
+                    .padding(.leading, -20.0)
+            }
+        }
+        .sheet(isPresented: $viewModel.showMicrophoneManualCalibation, content: {
+            MicrophoneManualCalibrationView { viewModel.showMicrophoneManualCalibation = false }
+        })
+    }
+    
+    private var autoCalibrateMicrophone: some View {
```

Comment:
Cane we make those two calibration settings buttons look and behave the same? Cause now one is a navigation ling and the other one makes a sheet appear and it looks a bit weird. Also indentation is still different for those two.

Reply:
I made them look the same, but I'll leave the modal vs navigation behavior since we don't really know if the auto calibration will stick